app/views/layouts/application.html.erb

```diff
@@ -1,17 +1,17 @@
 <!DOCTYPE html>
 <html>
+  <body class="site">
```

Comment:
This was in the correct spot. Head is outside of body. http://www.w3schools.com/tags/tag_head.asp